### PR TITLE
added mailing list

### LINKED ISSUE #101: [Site] Add links to GetNighthawk mailing lists
**Current Behavior**
Project mailing lists exist, but are not shown on the site.

**Desired Behavior**

These lists should be available for anyone to join:

- [Users](https://groups.google.com/a/getnighthawk.dev/g/users)
- [Developers(https://groups.google.com/a/getnighthawk.dev/g/developers)
- [Maintainers](https://groups.google.com/a/getnighthawk.dev/g/maintainers)

---

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by akashc777**
Signed-off-by: Akash Hadagali <akashc777@gmail.com>

**Description**

This PR fixes #101 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/_includes/mailing-list.html
Comment (DelusionalOptimist) [inline docs/_includes/mailing-list.html:6]: Nitpicking :P
```suggestion
            <p>Engage in the GetNightHawk project. Join any of the mailing lists.</p>
```

**Review by DelusionalOptimist — CHANGES_REQUESTED** [review]
Looks great @akashc777! :rocket: 
Just some very minor changes :D

**Comment by DelusionalOptimist** [comment]
The responsive one looks kinda odd :thinking: 


I don't know much about how UI works so feel free to ignore this if it is not in your control :slightly_smiling_face:

**Comment by DelusionalOptimist** [comment]
Also, you might need to do a rebase. [This](https://github.com/layer5io/meshery-adapter-library/blob/6db5d89b2022b73dc70ff7571dd53678e6aca98c/CONTRIBUTING-gitflow.md#submitting-your-changes-a-pull-request) might be useful for reference.

**Comment by akashc777** [comment]
@DelusionalOptimist I didn't receive designs for mobile view so didn't make it responsive yet 🙄

**Comment by DelusionalOptimist** [comment]
> @DelusionalOptimist I didn't receive designs for mobile view so didn't make it responsive yet roll_eyes

Oh! My bad. I should have seen it in the issue. :grimacing:

**Comment by akashc777** [comment]
I need some help with sign-off (sry I'm new to this)

**Comment by DelusionalOptimist** [comment]
Hmm :thinking:. Seems like applying suggestions from GitHub itself added `Co-authored-by: Rudraksh Pareek <rudrakshpareek3601@gmail.com>` with the commit message instead of your sign off in [`189d749` (#107)](https://github.com/layer5io/getnighthawk/pull/107/commits/189d749e5973d0b10fb61bba0d309c7be6e45ff2).
Maybe that's why DCO is failing.

**Comment by adithyaakrishna** [comment]
@akashc777 Could you run these commands, in the same order? I'm not sure they would fix it, but we can try this 👀 
- `git config --global user.name "Your Name"`
- `git config --global user.email "Your Email"` 
- `git commit --amend --no-edit --signoff`
- `git push --force-with-lease origin enhancement/mailing-list_section`

**Comment by akashc777** [comment]
@adithyaakrishna yup... 1st two are I think one-time thing... I did run the last two in the same order

**Comment by akashc777** [comment]
> Hmm 🤔. Seems like applying suggestions from GitHub itself added `Co-authored-by: Rudraksh Pareek <rudrakshpareek3601@gmail.com>` with the commit message instead of your sign off in [`189d749` (#107)](https://github.com/layer5io/getnighthawk/pull/107/commits/189d749e5973d0b10fb61bba0d309c7be6e45ff2).
> Maybe that's why DCO is failing.

i think that's the issue

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]
The mailing list component is merging with the footer. Both the footer and Mailing list components have the same background color. I am not sure if this is part of the design, but it looks odd (suggestions on this @austinogiza).
Also, the space occupied by the component is a bit larger than what we usually have for a mailing list ( for example, on this page https://layer5.io/projects/getnighthawk, both subscribe and cncf 'join us' components are taking a bit less space.)

**Comment by adithyaakrishna** [comment]
> @adithyaakrishna yup... 1st two are I think one-time thing... I did run the last two in the same order

Yeah, it's a one time thing, I thought there was an issue with the config regarding the signoffs

**Comment by akashc777** [comment]
> The mailing list component is merging with the footer. Both the footer and Mailing list components have the same background color. I am not sure if this is part of the design, but it looks odd (suggestions on this @austinogiza).
> Also, the space occupied by the component is a bit larger than what we usually have for a mailing list ( for example, on this page https://layer5.io/projects/getnighthawk, both subscribe and cncf 'join us' components are taking a bit less space.)

I did follow these designs https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=5884%3A12909
the mailing list buttons are bigger than the _join us_ button I think that's right?

File: docs/_includes/mailing-list.html
Comment (vinayaksh42) [inline docs/_includes/mailing-list.html:10]: Do a simple center alignment of all three buttons for the mobile view.

File: docs/_sass/mailing-list.scss
Comment (vinayaksh42) [inline docs/_sass/mailing-list.scss:57]: Remove this left margin for the mobile view using media queries

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]

**Comment by DelusionalOptimist** [comment]
So I'm listing some steps. Follow these to fix the DCO thingy:
* On your branch, do `git rebase -i d4c6596`
* In the editor that opens, change 
 ```
 pick 189d749 Update docs/_includes/mailing-list.html spelling mistakes
 ```
  to
 ```
 reword 189d749 Update docs/_includes/mailing-list.html spelling mistakes
 ```
 and save
* Another editor will open where you should remove the line that says
`Co-authored-by: Rudraksh Pareek <rudrakshpareek3601@gmail.com>` and save.
* Then you need to do
`git commit --amend -s` and save and quit. This should sign off this commit
* Check if your changes are alright and the history looks aligned.
* Force push to update changes here

Free tip, if you see vim `:w` to save `:q` to quit.

File: docs/_sass/mailing-list.scss
Comment (akashc777) [reply docs/_sass/mailing-list.scss:57]: I didn't receive designs for mobile view yet so it's not responsive

**Comment by vinayaksh42** [comment]
Try to follow this guide https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin for DCO issue

File: docs/_includes/mailing-list.html
Comment (akashc777) [reply docs/_includes/mailing-list.html:10]: you means to remove flex right?

**Comment by akashc777** [comment]
> So I'm listing some steps. Follow these to fix the DCO thingy:
> 
> * On your branch, do `git rebase -i d4c6596`
> * In the editor that opens, change
> 
> ```
> pick 189d749 Update docs/_includes/mailing-list.html spelling mistakes
> ```
> 
> to
> 
> ```
> reword 189d749 Update docs/_includes/mailing-list.html spelling mistakes
> ```
> 
> and save
> 
> * Another editor will open where you should remove the line that says
>   `Co-authored-by: Rudraksh Pareek <rudrakshpareek3601@gmail.com>` and save.
> * Then you need to do
>   `git commit --amend -s` and save and quit. This should sign off this commit
> * Check if your changes are alright and the history looks aligned.
> * Force push to update changes here
> 
> Free tip, if you see vim `:w` to save `:q` to quit.

Thanks a lot !
Now I'm not gonna use GitHub for future commits

**Comment by akashc777** [comment]
@DelusionalOptimist @vinayaksh42  made it responsive for mobile please have a look

**Review by DelusionalOptimist — APPROVED** [review]
This looks all good to me.
Nice work @akashc777 :tada:

**Review by vinayaksh42 — APPROVED** [review]
Great work @akashc777

**Comment by leecalcote** [comment]
Teamwork!

**Pull request merged**
